src/main/scala/sigmastate/serialization/DataJsonEncoder.scala

```diff
@@ -0,0 +1,160 @@
+package sigmastate.serialization
+
+import java.math.BigInteger
+import java.nio.charset.StandardCharsets
+
+import io.circe._
+import io.circe.syntax._
+import scalan.RType
+import sigmastate._
+import sigmastate.eval.{Colls, Evaluation, SigmaDsl}
+import sigmastate.lang.SigmaParser
+import sigmastate.lang.exceptions.SerializerException
+import special.collection.{Coll, collRType}
+import special.sigma.BigInt
+import spire.syntax.all.cfor
+
+import scala.collection.mutable
+
+object DataJsonEncoder {
+  implicit private val encodeBytes: Encoder[Array[Byte]] = (bytes: Array[Byte]) =>
+    Json.fromString(bytes.toString)
+
+  private def encodeName[T <: SType](tpe: T): String = tpe match {
```

Comment:
Misleading method name.
And actually there is SType.toTermString, which do exactly this.

Reply:
fixed